uitest/src/test/java/com/vaadin/tests/elements/notification/NotificationCloseEventTest.java

```diff
@@ -0,0 +1,40 @@
+package com.vaadin.tests.elements.notification;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.vaadin.testbench.elements.ButtonElement;
+import com.vaadin.testbench.elements.NotificationElement;
+import com.vaadin.tests.tb3.SingleBrowserTest;
+
+public class NotificationCloseEventTest extends SingleBrowserTest {
```

Comment:
Would it be possible to add a reloading test with `@PreserveOnRefresh` to the same UI and test class?

Reply:
Added